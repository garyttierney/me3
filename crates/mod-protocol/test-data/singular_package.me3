profileVersion = "v1"

[[package]]
id = "test-pkg"
path = "."
