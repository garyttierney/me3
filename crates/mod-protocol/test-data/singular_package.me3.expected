V1(
    ModProfileV1 {
        supports: [],
        natives: [],
        packages: [
            Package {
                id: Some(
                    "test-pkg",
                ),
                enabled: true,
                path: ModFile(
                    ".",
                ),
                load_after: [],
                load_before: [],
            },
        ],
    },
)
